src/sentry/eventstream/kafka/protocol.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import
+
+import pytz
+import logging
+from datetime import datetime
+
+from sentry.models import Event
+from sentry.utils import json
+
+
+logger = logging.getLogger(__name__)
+
+
+def handle_version_1_message(operation, event_data, task_state):
+    if operation != 'insert':
+        logger.debug('Skipping unsupported operation: %s', operation)
+        return None
+
+    event_data['datetime'] = datetime.strptime(
+        event_data['datetime'],
+        "%Y-%m-%dT%H:%M:%S.%fZ",
+    ).replace(tzinfo=pytz.utc)
```

Comment:
Stole this from https://github.com/getsentry/snuba/blob/e15c8c51140d55fa87772c150b62a11dc9bedee7/snuba/processor.py#L86